mlflow/server/auth/__init__.py

```diff
@@ -420,7 +420,6 @@ def _before_request():
         return
 
     _user = request.authorization.username if request.authorization else None
```

Comment:
this line can also go away

Reply:
Good catch, do you want to file a PR for that?